Tests/LegacyUnit/FrontEnd/DefaultControllerTest.php

```diff
@@ -1260,6 +1261,26 @@ public function singleViewCallsModifyEventSingleViewHook()
         $this->subject->main('', []);
     }
 
+    /**
+     * @test
+     */
+    public function singleViewCallsHookSeminarSingleViewModifySingleView()
+    {
+        $this->subject->setConfigurationValue('what_to_display', 'single_view');
+
+        /** @var \Tx_Seminars_Model_Event $event */
+        $event = \Tx_Oelib_MapperRegistry::get(\Tx_Seminars_Mapper_Event::class)->find($this->seminarUid);
+        $hookedInObject = $this->createMock(SeminarSingleView::class);
+        $hookedInObject->expects(self::once())->method('modifySingleView')->with($this->subject);
+
+        $hookedInClass = get_class($hookedInObject);
```

Comment:
Please add a backslash before `get_class` to improve performance (once this test gets namespaced).